setup.py

```diff
@@ -15,7 +15,7 @@
 
 setuptools.setup(
     name="netpyne_ui",
-    version="0.7.0",
+    version="0.8.0",
```

Comment:
the version change should be a release branch thing. If this feature branch is derived from the release branch, it should be merged back to the release branch

Reply:
Rolled back as requested